<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewWidget</class>
 <widget class="QWidget" name="NewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Head" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>901</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="Index" name="page_3"/>
   <widget class="Tree_view" name="page"/>
   <widget class="Set" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="set_btn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="monitor_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>540</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="index_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Head</class>
   <extends>QWidget</extends>
   <header>head.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tree_view</class>
   <extends>QWidget</extends>
   <header>tree_view.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Set</class>
   <extends>QWidget</extends>
   <header>set.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Index</class>
   <extends>QWidget</extends>
   <header>index.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
